.ci/macos-steps.yml

```diff
@@ -1,3 +1,6 @@
+parameters:
```

Comment:
no need for this imo

Reply:
true, I just didn't know if it was needed to capture the env variable, not sure why it isn't working :(